internal/appsec/_tools/libddwaf-updater/update.sh

```diff
@@ -59,14 +80,30 @@ curl -L https://github.com/DataDog/libddwaf/releases/download/$version/libc++-st
 #  object file by using ld -r
 run_binutils x86_64-linux-gnu-ld \
    -r -o $bindings_dir/lib/linux-amd64/libddwaf.a \
-   --require-defined=ddwaf_init \
-   --require-defined=ddwaf_get_version \
-   --require-defined=ddwaf_destroy \
-   --require-defined=ddwaf_context_init \
-   --require-defined=ddwaf_result_free \
```

Comment:
Is ddwaf_result_free not part of the API anymore?

Reply:
Yes it is still used in this version indeed. I reverted it. The symbol was still defined probably due to one of the others using it.